etc/encoding/opencast-movies.properties

```diff
@@ -168,3 +168,19 @@ profile.import.silent.input = nothing
 profile.import.silent.output = audio
 profile.import.silent.suffix = -silent-audio.mp4
 profile.import.silent.ffmpeg.command = -strict -2 -filter_complex aevalsrc=0::d=#{time} -c:a aac -b:a 8k #{out.dir}/#{out.name}#{out.suffix}
+
+# Transcode videos generated with opencast studio to cover some issues
+# The video resolution should be reduced to maximum of 1080p.
```

Comment:
It makes no sense to limit this to 1080p here but then include 2.5k and 4k encoding later.